docs/sources/source/nlp/ner.rst

```diff
@@ -12,115 +12,120 @@ This tutorial explains how to implement named entity recognition (NER) in NeMo.
 Download Dataset
 ----------------
 
-`CoNLL-2003`_ is a standard evaluation dataset for NER, but any NER dataset will work. The only requirement is that the files are formatted like this:
+`CoNLL-2003`_ is a standard evaluation dataset for NER, but any NER dataset will work. CoNLL-2003 dataset could also be found `here`_. The only requirement is that the data is splitted into 2 files: text.txt and labels.txt. The text.txt files should be formatted like this:
 
 .. _CoNLL-2003: https://www.clips.uantwerpen.be/conll2003/ner/
+.. _here: https://github.com/kyzhouhzau/BERT-NER/tree/master/data
 
 .. code-block::
 
-    Jennifer	B-PER
-    is		O
-    from	O
-    New		B-LOC
-    York	I-LOC
-    City	I-LOC
-    .		O
+    Jennifer is from New York City.
+    She likes ...
+    ...
+
+The labels.txt files should be formatted like this:
+
+.. code-block::
 
-    She		O
-    likes	O
+    B-PER O O B-LOC I-LOC I-LOC O
+    O O ...
     ...
 
-Here, the words and labels are separated with spaces, but in your dataset they should be separated with tabs. Each line should follow the format: [WORD] [TAB] [LABEL] (without spaces in between). There can be columns in between for part-of-speech tags, as shown on the `CoNLL-2003 website`_. There should also be empty lines separating each sequence, as shown above.
+Each line of the text.txt file contains text sequences, where words are separated with spaces. The labels.txt file contains corresponding labels for each word in text.txt, the labels are separated with spaces. Each line of the files should follow the format: [WORD] [SPACE] [WORD] [SPACE] [WORD] (for text.txt) and [LABEL] [SPACE] [LABEL] [SPACE] [LABEL] (for labels.txt). There can be columns in between for part-of-speech tags, as shown on the `CoNLL-2003 website`_.
 
 .. _CoNLL-2003 website: https://www.clips.uantwerpen.be/conll2003/ner/
 
-.. _Preprocessed data: https://github.com/kyzhouhzau/BERT-NER/tree/master/data
+
 
 Training
 --------
 
 .. tip::
 
     We recommend you try this out in a Jupyter notebook. It'll make debugging much easier!
+    See examples/nlp/NERWithBERT.ipynb
 
 First, we need to create our neural factory with the supported backend. How you should define it depends on whether you'd like to multi-GPU or mixed-precision training. This tutorial assumes that you're training on one GPU, without mixed precision. If you want to use mixed precision, set ``amp_opt_level`` to ``O1`` or ``O2``.
 
```

Comment:
explain that if we don't want to use mixed-precision, use ``O0``

Reply:
done